pkg/models/jwt.go

```diff
@@ -0,0 +1,31 @@
+package models
```

Comment:
We're changing our package hierarchy philosophy, see https://github.com/grafana/grafana/blob/master/contribute/architecture/backend/package-hierarchy.md. 

This should rather be in the pkg/services/auth/jwt package and perhaps the implementation in pkg/services/auth/jwt/service. Something that could be addresses later (after merging this).